test/unit/material_container_test.cpp

```diff
@@ -0,0 +1,11 @@
+#include "../../include/catch.hpp"
```

Comment:
You don't need to make this file. firstly it's empty, and secondly we already have a test binary being created that you can latch onto. Look at the CMakeLists.txt for where to add your test files.